HttpInspect.as

```diff
@@ -27,7 +27,7 @@ RequestInfo@ AddRequestInfo(const string &in method)
 
 string CommandLineSafe(const string &in str)
 {
-	return str.Replace("\\", "\\\\").Replace("\"", "\\\"");
+	return str.Replace("\\", "\\\\").Replace("'", "'\''";
```

Comment:
The last string should be `"\\'"`, and you're missing a closing `)`.

Reply:
Imagine getting called out for thinking you can get away with doing minor changes in the web editor.. Couldn't be me.

Last string should be `"'\\''"`. Forgot to escape the backslash. The other single quotes are needed to end/start the literal:
```
$ echo 'foo'bar'
> '     # string in last line was not closed with an '
foobar

$ echo 'foo'\''bar'
foo'bar
```
Gotta love bash.

Reply:
Gotta love bash also means this totally might break on other shells (e. g. zsh or fish) - but i think this is out of scope, you'd need a dedicated sub menu to support all the shells any syntaxes.